components/org.wso2.carbon.identity.local.auth.api.endpoint/pom.xml

```diff
@@ -87,6 +87,12 @@
     </build>
 
     <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.governance</groupId>
+            <artifactId>org.wso2.carbon.identity.user.session</artifactId>
+            <scope>provided</scope>
+            <version>1.0.1</version>
```

Comment:
Move the version as a property to the parent pom

Reply:
Resolved with 78b6e0a